src/__tests__/act.test.tsx

```diff
@@ -1,12 +1,12 @@
-// @flow
 import * as React from 'react';
 import { Text } from 'react-native';
 import ReactTestRenderer from 'react-test-renderer';
 import act from '../act';
 import render from '../render';
 import fireEvent from '../fireEvent';
 
-const UseEffect = ({ callback }: { callback: Function }) => {
+type UseEffectProps = { callback: () => void };
```

Comment:
small improvement (?):

```suggestion
type UseEffectProps = { callback(): void };
```